packages/test/src/test-workflows.ts

```diff
@@ -800,24 +804,24 @@ test('failSignalWorkflow', async (t) => {
   }
   {
     const req = cleanWorkflowFailureStackTrace(await activate(t, await makeSignalWorkflow('fail', [])));
+    const stackTrace = removeWorkflowFailureStackTrace(req);
     compareCompletion(
       t,
       req,
       makeSuccess(
-        [
-          makeFailWorkflowExecution(
-            'Signal failed',
-            dedent`
-          ApplicationFailure: Signal failed
-              at Function.nonRetryable (common/src/failure.ts)
-              at test/src/workflows/fail-signal.ts
-          `,
-            'Error'
-          ),
-        ],
+        [makeFailWorkflowExecution('Signal failed', 'Error')],
```

Comment:
What if we kept the reference stack trace as argument to `makeFailWorkflowExecution()`, and instead did the special handling of stack traces as part of `compareCompletion()`? We'd simply iterate over history events, looking for `WorkflowExecutionFailure` events (if any). WDYT?